### PR TITLE
Create a view for Categories, Subcategories, Items

### LINKED ISSUE #64: Show Categories, Subcategories, Items in the admin panel under AcademiX
**Is your feature request related to a problem? Please describe.**
- We need to show the Categories, Subcategories, Items in the admin panel under AcademiX

**Describe the solution you'd like**
- Create a view for Categories, Subcategories, Items

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #64 

## Goals

-  Update admin panel to add/edit Categories, Subcategories, Items

## Approach

- Add Categories view
- Add subcategories view
- Add items view
- Update interfaces accordingly to the new DB structure  
phase

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/styles.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:local .content {
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/styles.css:1]: no need to add `:local` prefix. Every CSS class is `:local` by default.

File: frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/interfaces.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  items: Item[];
+  isLoading: boolean;
+  pagination: {
+    current: number,
+    pageSize: number,
+    total: number,
+  };
+}
+
+export interface Item {
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/interfaces.ts:11]: The item interface is already defined in home. Use the same.

File: frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/styles.css (hunk 1 above)
Comment (anjula-sack) [reply frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/styles.css:1]: ok

File: frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/interfaces.ts (hunk 2 above)
Comment (anjula-sack) [reply frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/interfaces.ts:11]: ah yes.

**Pull request merged**